cves/kernel/CVE-2019-19057.yml

```diff
@@ -238,7 +248,7 @@ interesting_commits:
       * Anything else you find interesting.
   commits:
   - commit:
-    note:
+    note: No commits between the VCC and the fix commits
```

Comment:
I ran git log --stat  0732484b47b57ef90bb08408d60fddbad0262d82..d10dcb615c8e29d403a24d35f8310a7a53e3050c -- drivers/net/wireless/marvell/mwifiex/pcie.c and saw some commits. Not sure if they are interesting enough.